app/src/main/res/layout/options_display.xml

```diff
@@ -38,6 +38,13 @@
                 android:layout_height="wrap_content"
                 android:orientation="vertical">
 
+                <com.igalia.wolvic.ui.views.settings.RadioGroupSetting
+                    android:id="@+id/homepage"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    app:description="@string/developer_options_homepage"
+                    app:options="@array/developer_options_homepage" />
```

Comment:
This is not a blocker but I am not sure about this UI, looks weird to me having this horizontal radio selector that might unveil a homepage editor UI if one of the options is selected. 

I think we should keep the current UI and just add a switch that would enable "new tab" and hide the homepage selection UI if enabled

Reply:
Yes, it might not be the best UI for this choice. I will improve it in a subsequent PR.